exasol/ds/sandbox/lib/asset_printing/print_assets.py

```diff
@@ -147,6 +147,7 @@ def print_export_image_tasks(aws_access: AwsAccess, filter_value: str, printing_
     text_print.print(tuple())
 
 
+# TODO: Clarify if objects in Example Data S3 buckets should be printed, too
```

Comment:
No, assets are part of a release, example data is not

Reply:
Thanks! Comment is removed in next push.